ws-nextjs-app/cypress/e2e/livePage/index.cy.ts

```diff
@@ -11,7 +12,24 @@ const testDetails = {
       id: 'c7p765ynk9qt',
       runforEnv: ['test', 'local'],
       service: 'pidgin',
-      tests: [testsThatAlwaysRunForAllPages, pageVisit, mediaPlayerTests],
+      tests: [
+        testsThatAlwaysRunForAllPages,
+        pageVisit,
+        mediaPlayerTests,
+        keyPoints,
+      ],
+    },
+    {
+      path: '/urdu/live/cx2qdkezzzvt',
+      id: 'cx2qdkezzzvt',
+      runforEnv: ['test', 'live'],
```

Comment:
This asset won't work on test
```suggestion
      runforEnv: ['live'],
```